apps/performance-tests/src/Datasets.ts

```diff
@@ -4,8 +4,54 @@
  *--------------------------------------------------------------------------------------------*/
 import fs from "fs";
 import path from "path";
+import { insertPhysicalElement, insertPhysicalModelWithPartition, insertSpatialCategory } from "presentation-test-utilities/lib/IModelUtils";
+import { createIModel } from "./util/IModelUtilities";
 
-async function downloadDataset(name: string, downloadUrl: string, localPath: string): Promise<void> {
+const LARGE_FLAT_IMODEL_SIZE = 50_000;
+const BAYTOWN_DOWNLOAD_URL = "https://github.com/imodeljs/desktop-starter/raw/master/assets/Baytown.bim";
+
+export class Datasets {
+  private static readonly _iModels: { baytown?: string; largeFlat?: string } = {};
+
+  public static get bayTown(): string {
+    return this.verifyInitialized(this._iModels.baytown);
+  }
```

Comment:
I'm not sure we want to add this getter for each dataset. Maybe we should have something like this?
```ts
type IModelNames = "baytown" | "50k elements";
type IModelPathsMap = {[name in IModelNames]: string};
class Datasets {
  private static _map: IModelPathsMap;
  public static getPath(name: IModelNames) {
    return this.verifyInitialized(this._map[name]);
  }
}
```

Reply:
Change it to a typed map